SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp

```diff
@@ -161,8 +167,11 @@ bool FSendDeleteEntityRequest::Update()
 {
 	const Worker_EntityId EntityId = 0;
 	USpatialWorkerConnection* Connection = ConnectionManager->GetWorkerConnection();
-	Connection->SendDeleteEntityRequest(EntityId, RETRY_UNTIL_COMPLETE);
-	Connection->Flush();
+	if (Connection)
```

Comment:
If Connection = null should this return false or fail?

Reply:
Explicit check here?

Reply:
Sure, it will just timeout in the current version.